web-app-router/scenes/Portal/Teams/TeamId/Team/common/TeamProfile/index.tsx

```diff
@@ -0,0 +1,67 @@
+"use client";
+
+import { useFetchTeamLazyQuery } from "./graphql/client/fetch-team.generated";
+import { Fragment, useEffect } from "react";
+import { useParams } from "next/navigation";
+import { TYPOGRAPHY, Typography } from "@/components/Typography";
+import { UserMultipleIcon } from "@/components/Icons/UserMultipleIcon";
+import { Image } from "./Image";
+import Skeleton from "react-loading-skeleton";
+import { twMerge } from "tailwind-merge";
+import clsx from "clsx";
+
+export const TeamProfile = (props: { className?: string }) => {
+  const [fetchTeam, { data }] = useFetchTeamLazyQuery();
+  const { teamId } = useParams() as { teamId: string };
+
+  useEffect(() => {
+    if (!teamId) {
+      return;
+    }
+
+    fetchTeam({
+      variables: { teamId },
+      context: { headers: { team_id: teamId } },
+    });
+  }, [fetchTeam, teamId]);
+
+  return (
+    <div
+      className={twMerge(
+        clsx(
+          "grid grid-cols-auto/1fr grid-rows-2 gap-x-5 gap-y-2 justify-self-start py-8 border-b border-dashed border-grey-200",
+          props.className,
+        ),
+      )}
+    >
+      <div className="row-span-2 w-20 h-20 rounded-2xl overflow-hidden">
+        {data?.team_by_pk?.name && (
+          // FIXME: Pass the correct src
+          <Image src={null} teamName={data?.team_by_pk?.name} alt="Team logo" />
```

Comment:
we do not have images for this and will be deprioritizes since this will require hasura changes as well